envoy/datadog_checks/envoy/data/conf.yaml.example

```diff
@@ -75,33 +109,44 @@ instances:
     #
     # cache_metrics: true
 
-    ## @param username - string - optional
-    ## The username to use if services are behind basic auth.
-    ## Note: The Envoy admin endpoint does not support auth until:
-    ## https://github.com/envoyproxy/envoy/issues/2763
-    ## For an alternative, see:
-    ## https://gist.github.com/ofek/6051508cd0dfa98fc6c13153b647c6f8
+    ## @param tags - list of strings - optional
+    ## A list of tags to attach to every metric and service check emitted by this instance.
+    ##
+    ## Learn more about tagging at https://docs.datadoghq.com/tagging
     #
-    # username: <USERNAME>
+    # tags:
+    #   - <KEY_1>:<VALUE_1>
+    #   - <KEY_2>:<VALUE_2>
 
-    ## @param password - string - optional
-    ## The password to use if services are behind basic or NTLM auth.
-    ## Note: The Envoy admin endpoint does not support auth until:
-    ## https://github.com/envoyproxy/envoy/issues/2763
-    ## For an alternative, see:
-    ## https://gist.github.com/ofek/6051508cd0dfa98fc6c13153b647c6f8
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Overrides any `service` defined in the `init_config` section.
     #
-    # password: <PASSWORD>
+    # service: <SERVICE>
+
+    ## @param min_collection_interval - number - optional - default: 15
+    ## This changes the collection interval of the check. For more information, see:
+    ## https://docs.datadoghq.com/developers/write_agent_check/#collection-interval
+    #
+    # min_collection_interval: 15
 
-    ## @param proxy - object - optional
+    ## @param empty_default_hostname - boolean - optional - default: false
+    ## This forces the check to send metrics with no hostname.
+    ##
+    ## This is useful for cluster-level checks.
+    #
+    # empty_default_hostname: false
+
+    ## @param proxy - mapping - optional
     ## This overrides the `proxy` setting in `init_config`.
     ##
-    ## Set HTTP or HTTPS proxies. Use the `no_proxy` list
+    ## Set HTTP or HTTPS proxies for all instances. Use the `no_proxy` list
```

Comment:
This is incorrect, it just sets the proxy to the current instance. But this should be fixed in a separate PR.